File: POO/Exo1/Implementation C++/Complement.cpp
#include <iostream>
#include "Complement.hpp"

using namespace std;

Complement::Complement(string NomComplement,int QteComplement,string CouleurComplement)
	{
		mNomComplement=NomComplement;
		mQteComplement=QteComplement;
		mCouleurComplement=CouleurComplement;
	}
string Complement::AfficherNomComplement()
	{
		return mNomComplement;
	}
int Complement::AfficherQteComplement()
	{
		return mQteComplement;
	}
string Complement::AfficherCouleurComplement()
	{
		return mCouleurComplement;
	}

File: POO/Exo1/implementation java/Complement.java
public class Complement{
	private String mNomComplement;
	private int mQteComplement;
	private String mCouleurComplement;
	public Complement(String NomComplement,int QteComplement,String CouleurComplement)
	{
		this.mNomComplement=NomComplement;
		this.mQteComplement=QteComplement;
		this.mCouleurComplement=CouleurComplement;
	}
	public String AfficherNomComplement()
	{
		return this.mNomComplement;
	}
	public int AfficherQteComplement()
	{
		return this.mQteComplement;
	}
	public String AfficherCouleurComplement()
	{
		return this.mCouleurComplement;
	}
}
